build.rs

```diff
@@ -5,6 +5,16 @@ use std::process::Command;
 fn main() {
     #[cfg(feature = "llvm")]
     {
+        Command::new("make")
+            .args(["-C", "stdlib", "solana_stdlib"])
+            .output()
+            .expect("Could not build Solana stdlib");
+
+        Command::new("make")
+            .args(["-C", "stdlib", "substrate_stdlib"])
+            .output()
+            .expect("Could not build Substrate stdlib");
+
```

Comment:
Why is this separated out into two commands?

Reply:
As I explained in your previous comment, I couldn't get `make all` to build everything.